docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,107 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods.
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, there are some pods other than ScalarDB Cluster pods on the worker nodes.
+
+In other words, the following components run on one worker node:
+
+* ScalarDB Cluster pod (2vCPU / 4GB)
+* Envoy proxy if you use `indirect` client mode (0.2–0.3 vCPU / 256–328 MB)
+* Kubernetes components
+* Your application pod
+
+**Note:** You do not need to deploy an Envoy pod when using `direct-kubernetes` mode.
+
+With this in mind, you should use a worker node that has at least 4vCPU / 8GB memory resources and use at least three worker nodes for availability that we mentioned in the previous section [Number of pods and Kubernetes worker nodes](./ProductionChecklistForScalarDBCluster.md#number-of-pods-and-kubernetes-worker-nodes).
+
+You should also consider the worker node resources based on the resources that your application pod use. If you plan to scale the pods automatically by using some features like [Horizontal Pod Autoscaling (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/), you should also consider the maximum number of pods on the worker node.
+
+### Network
+
+You should create the Kubernetes cluster on a private network since ScalarDB Cluster does not provide any services to users directly via internet access. We recommend accessing ScalarDB Cluster via a private network from your applications.
+
+### Monitoring and logging
+
+You should monitor the deployed components and collect their logs. For details, see [Monitoring Scalar products on a Kubernetes cluster](./K8sMonitorGuide.md) and [Collecting logs from Scalar products on a Kubernetes cluster](./K8sLogCollectionGuide.md).
+
+### Backup and restore
+
+You should enable the automatic backup feature and PITR feature in the backend database. For details, see [Set up a database for ScalarDB/ScalarDL deployment](./SetupDatabase.md).
+
+## Production checklist: Client applications that access ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up a client application that accesses ScalarDB Cluster in a production environment.
+
+### Client mode
```

Comment:
Apologies for overlooking this in the previous review. The client mode and configurations are specifically for the ScalarDB Java Client we provide. If users interact with the gRPC API and write their own client code, these settings won't apply. We should probably highlight this.

Reply:
Ah, I see. That's right. We should add a note about gRPC (not Java library) pattern. I will fix it. Thank you!

Reply:
@brfrn169 
I updated the description of client mode in 65819854f1692b7b04421c12c0e3984b662736ab.
Please take a look when you have time!

Reply:
Thank you for the modification!

I think the entire section titled `Production checklist: Client applications that access ScalarDB Cluster` is only for the Java Client library. So probably, we should change the section name to `Production checklist: Client applications that access ScalarDB Cluster (Java client library only)`?